configure

```diff
@@ -1128,7 +1136,7 @@ if [[ $USECUDA -eq 1 ]] ; then
     echo "Error: nvcc cuda compiler not found in $CUDA_HOME/bin/" > /dev/stderr
     exit 1
   fi
-  if [ -z "$SHADER_MODEL" ] ; then
+  if [ -z "$NVCCFLAGS" -a -z "$SHADER_MODEL" ] ; then
```

Comment:
The `SHADER_MODEL` variable here is very limited.  As far as I could tell, it restricts CUDA compilation to a single h/w revision (for Jenkins, that's SM 3.0, since that's what my GTX 680 is).

By contrast, the Amber config.h file has nvcc flags passed to pmemd.cuda that allow it to execute with *any* supported h/w revision.  I recommend adopting that same logic here (unless that's what `NVCCFLAGS` was added for).